src/index.js

```diff
@@ -59,9 +65,19 @@ function IpfsAPI (host_or_multiaddr, port) {
 
   // -- Interface
 
+  /** @public */
   self.send = requestAPI
 
-  self.add = (files, opts, cb) => {
+  /**
+   * Add a file/many files to IPFS returning the hash and name. The
+   * name value will only be set if you are actually sending a file.
+   *
+   * @public
+   * @param {} files
+   * @param {object} opts
+   * @param {function} cb
+   */
```

Comment:
Jsdoc is pretty sweet, I just get annoyed at it because you end up with truly massive js files. It definitely helps out documentation, though, and I've used it before. 

The only real con is that updating it can get old. but not as old as old docs.